### PR TITLE
Dread: Clean up Artaria and Ghavoran logic

### PR DESCRIPTION
**Pull request by XenoWars**
- Changes a bunch of things that had the option of Walljump (Beginner) to trivial
- Changed some Grapple options to include Grapple Movement
- Changed some Movement tricks to Climb Sloped Tunnels
- Changed some Movement tricks to Skip Cross Bomb

### CODE REVIEW

File: randovania/games/dread/json_data/Ghavoran.txt [hunk 1]
```diff
@@ -691,10 +697,11 @@ Extra - asset_id: collision_camera_009
   * Extra - right_shield_def: actordef:actors/props/doorshieldmissile/charclasses/doorshieldmissile.bmsad
   > Bomb Ledge
       Any of the following:
-          Grapple Beam or Space Jump or Simple IBJ
+          Space Jump or Simple IBJ or Use Spin Boost
```
Comment (Hugoshido) [inline randovania/games/dread/json_data/Ghavoran.txt:700]: Sure this isn´t trivial?

**Review by Hugoshido — COMMENTED** [review]
One question

File: randovania/games/dread/json_data/Ghavoran.txt (hunk 1 above)
Comment (XenoWars) [reply randovania/games/dread/json_data/Ghavoran.txt:700]: Yes, you have to use spin boost to wall jump to the ledge

**Review by Hugoshido — APPROVED** [review]

File: randovania/games/dread/json_data/Ghavoran.txt (hunk 1 above)
Comment (henriquegemignani) [reply randovania/games/dread/json_data/Ghavoran.txt:700]: `Use Spin Boost` covers having Space Jump.

**Pull request merged**